README.md

```diff
@@ -13,5 +13,8 @@ It uses a pretrained version of a BiT model to encode an image from an array of
 The following parameters can be used:
 
 - `model_path` (string, default: "pretrained"): The folder where the downloaded pretrained Big Transfer model is located
+- `model_name` (string, default: "R50x1"): The model to be downloaded when the model_path is empty. Choose from ['R50x1', 'R101x1', 'R50x3', 'R101x3', 'R152x4']
```

Comment:
For the README, we could refer to the template to have the same contents structure

Reply:
https://github.com/jina-ai/executor-big-transfer-encoder/pull/1/commits/b845ac37e6f21776c11dbcfe17be6573e30e0208